website/client/src/components/shops/balanceInfo.vue

```diff
@@ -67,33 +71,40 @@ export default {
   computed: {
     currencies () {
       const currencies = [];
```

Comment:
this is so weird, why do we make an array and push stuff into it instead of just defining the array in the const declaration? or one better, defining the possible currencies in the common code... :thinking: You don't necessarily need to refactor it, just .. huh

Reply:
I rewrote it so the array is defined within the const declaration instead using `.push()`